config/initializers/new_framework_defaults_6_1.rb

```diff
@@ -33,16 +33,16 @@
 
 # Specify whether `ActiveSupport::TimeZone.utc_to_local` returns a time with an
 # UTC offset or a UTC time.
-# ActiveSupport.utc_to_local_returns_utc_offset_times = true
+ActiveSupport.utc_to_local_returns_utc_offset_times = true
```

Comment:
We don't use `utc_to_local` anywhere in the app, thankfully. I've only ever had to use it in one app (because Rails co-opts the TZ-less type in Postgres to mean UTC :sob:) and it was a nightmare to keep track of.